docs/reference/modules/discovery/adding-removing-nodes.asciidoc

```diff
@@ -12,12 +12,17 @@ cluster, and to scale the cluster up and down by adding and removing
 master-ineligible nodes only. However there are situations in which it may be
 desirable to add or remove some master-eligible nodes to or from a cluster.
 
-==== Adding master-eligible nodes
+==== Adding nodes
```

Comment:
You're absolutely right that the words in this section apply to master-ineligible nodes too, and we can therefore remove the `master-eligible` in the next line as well.

However if we make this change then it becomes obvious that we are missing documentation about removing master-ineligible nodes. Perhaps this is the right place to write about that too, but that needs some thought. I would prefer not to make this change now and to follow-up later.